src/components/Cards.jsx

```diff
@@ -0,0 +1,87 @@
+              fill="#FFE81F"
+            ></path>
+            <g fill="#FFE81F">
+              <path
+                className="st2"
+                d="M326.5 53.9v28.6h-57.3V170h-34.1V82.5l-66.9.2c-8.1 0-9.1 6.4-9.1 8.9 0 3 2 6 12.8 19.7 7 8.9 13.5 17.8 14.3 19.7 4.6 10.7-1.4 27.3-12.3 34.1-7.8 4.8-6.5 4.9-64.3 4.9H37.9v-32.1h97.9l2.9-2c1.9-1.4 3.1-3.2 3.5-5.8.6-3.7.3-4.2-11.8-17.8-14.3-16.1-15.8-19-15.1-29.3.8-11.7 11.3-28.8 29.6-28.8l181.6-.3m8-8h-8l-181.6.3c-10.5 0-20.2 4.5-27.3 12.7-5.8 6.7-9.6 15.4-10.2 23.5-1 13.5 2.1 18.4 17.1 35.2 4.3 4.9 8.4 9.5 9.8 11.5 0 .1 0 .1-.1.2l-.1.1-.8.6H29.9v48h79.7c28.5 0 42.6 0 50.9-.6 9.5-.7 12.5-2.3 17.1-5.1l.6-.4c14.3-8.8 21.6-29.8 15.4-44.1-1.9-4.4-14-19.7-15.4-21.5-3.8-4.8-10.1-12.7-11.1-15 0-.2.1-.4.1-.5.2 0 .5-.1 1-.1l58.9-.2V178h50.2V90.5h57.3V45.9h-.1zM419.5 54.9l40.2 116.2-32.8-.2-5.3-17.4-58.4-.3L358 170h-33.4l39.7-115.1h55.2m-46.8 73.6l39.8-.1c.1 0-19.3-57.2-19.5-57.2-.1 0-20.7 57.3-20.3 57.3m52.5-81.6H358.6l-1.9 5.4L317 167.4l-3.7 10.6h50.6l1.8-5.6 3.5-11.1 46.6.2 3.6 11.8 1.7 5.6h5.9l32.8.2 11.3.1-3.7-10.7-40.3-116.3-1.9-5.3zm-41.3 73.6c2.5-7 5.9-16.7 9-25.3 2.9 8.6 6.2 18.2 8.6 25.3h-17.6zM532.4 54.9c25.9 0 43.9.4 47.4 1 8 1.4 15.2 5.7 20.7 12.2 5.7 6.9 7 10.1 7.7 19.5 1.1 13.3-5.2 24.7-17.7 32.1-3.6 2.2-7.4 4.3-8.3 4.7-1.4.6-1.2 1.3 1.3 4.3 1.6 1.9 4.5 4.7 6.3 6.1l3.4 2.6 62.1.9.6 31.7h-38.1c-29.4 0-38.9-.3-41.6-1.2-4.1-1.4-14.1-9.8-34.3-29L527.5 126l.2 44h-37V54.9h41.7m-5.5 51.7h19.7c19.1 0 19.7-.1 22.5-2.1 4.9-3.7 6.6-7.4 6.1-13.2-.5-6-2.8-9.1-8.6-11.6-3.2-1.4-6.4-1.7-21.7-1.7H527v28.6m5.4-59.7h-49.8V178h53v-8l-.1-25.2.8.8c25 23.8 32.4 29.2 37.2 30.8 3.7 1.3 10 1.6 44.2 1.6H664l-.2-8.2-.6-31.7-.1-7.7-7.7-.1-59.4-.9-1.2-1c-.4-.3-.9-.7-1.4-1.2.4-.3.9-.5 1.2-.7 15-9 22.9-23.4 21.6-39.7-.9-11-2.8-15.9-9.5-24-6.6-8-15.6-13.3-25.4-15-5.6-.8-33-.9-48.9-.9zm2.5 39.2h9.9c14 0 16.8.2 18.5 1 3.2 1.4 3.6 2.1 3.8 5 .3 2.8-.1 4-2.9 6.1-.1.1-.2.1-.2.2-1.9.3-8.8.3-17.4.3h-11.7V86.1z"
+                fill="#FFE81F"
+              ></path>
+            </g>
+          </g>
+        </svg>
+      </header>
+      {loading && <div>Data is loading. Please wait...</div>}
+      {error && <div>{`There is a problem fetching your data - ${error}`}</div>}
+      <ul>
+        {data &&
+          data.map((movie, index) => {
+            const bgIndex = index % BackgroundUrls.length;
```

Comment:
What does this line do?

Reply:
`const bgUrl = BackgroundUrls[index];`

Reply:
It creates an id for each background image, just a way to escape the error message in the console, when index is used.